### ISSUE TITLE
Error: Cannot find module 'autoprefixer-core'

### ISSUE THREAD

**Issue by Frikki** [nDmitry/grunt-postcss#3]
In my project, I run the following:

```
npm install grunt-autoprefixer --save-dev
npm install grunt-postcss --save-dev
```

When I add the following to my `Gruntfile.js` file from the usage example:

```
grunt.initConfig({
  postcss: {
    options: {
      processors: [
        require('autoprefixer-core')({browsers: 'last 1 version'}).postcss
      ]
    },
    dist: {
      src: 'css/*.css'
    }
  }
});
```

...I get: `Error: Cannot find module 'autoprefixer-core'`.

Platform: Windows 7
NodeJS v0.10.18
npm 1.3.8
grunt-cli v0.1.13
grunt v0.4.5

**Comment by giovannipds** [comment]
Hello!

Sorry for opening this issue again, but I'm having the same as metioned by @Frikki... BUT, I'm already using autoprefixer-core lol.

What I noticed is that, any "require()" doesn't find the module specified...  any ideas?

Cheers!

**Comment by giovannipds** [comment]
Ok, I came back to answer myself.

The Grunt example at http://github.com/postcss/autoprefixer/blob/master/README.md#grunt was previously wrong, and I was guided by it. Today, @renestalder fixed it, at postcss/autoprefixer@bf85b68ba9b440b7a372b6b1d820e184c4ee7738. Thanks @renestalder.

Another thing that I was doing wrong, is that, I was trying to load http://github.com/postcss/autoprefixer-core with `grunt.loadNpmTasks('autoprefixer-core');` (WRONG) without needing. Inside of the postcss configuration you just need to use the:

    require('autoprefixer-core')({
        browsers: ['last 2 versions']
    })

And everything will work like a charm.

If anyone land here, I hope the comment help to make it through. Best wishes!

**Comment by nDmitry** [comment]
This feature seems to be popular in search results, so I will leave here a link to our migration guide from grunt-autoprefixer to grunt-postcss: https://github.com/nDmitry/grunt-postcss#how-to-migrate-from-grunt-autoprefixer
